cli/src/commands/bookmark/set.rs

```diff
@@ -124,3 +128,43 @@ pub fn cmd_bookmark_set(
     )?;
     Ok(())
 }
+
+fn check_has_at_symbol(ui: &mut Ui, bookmark_names: &[String]) -> Result<(), CommandError> {
+    let has_at = bookmark_names
+        .iter()
+        .filter(|name| {
+            let Some((left, right)) = name.split_once('@') else {
+                return false;
+            };
+            !(left.is_empty() && right.is_empty())
+        })
+        .collect_vec();
+
+    if !has_at.is_empty() {
+        let is_plural = has_at.len() > 1;
+        let mut writer = ui.warning_with_heading(if is_plural {
+            "Suspicious bookmark names:"
+        } else {
+            "Suspicious bookmark name:"
+        });
+        if has_at.len() > 1 {
+            writeln!(writer)?;
+        }
+        for name in &has_at {
+            writeln!(writer, "  {name}")?;
+        }
+        drop(writer);
```

Comment:
nits: 
* just use `ui.warning_default()` to print "Warning: "?
* we generally avoid ad-hoc singular/plural handling. just use plural form.
* print bookmarks in line (with `.join(", ")`)? it's unlikely there are bunch of bookmarks to be created

Reply:
done